### PR TITLE
Fix/storage explorer get all ttems along folder limit

### PR DESCRIPTION
**Pull request by joshenlim**
- Optimize /list limit in `getAllItemsAlongFolder` to reduce requests + reduce time taken to retrieve files
- Fix FileExplorerRow stale name when creating folder (it uses the first folder's name in the input field)

### CODE REVIEW

File: studio/localStores/storageExplorer/StorageExplorerStore.js [hunk 1]
```diff
@@ -1572,6 +1576,8 @@ class StorageExplorerStore {
   }
 
   addTempRow = (type, name, status, columnIndex, metadata = {}, isPrepend = false) => {
+    console.log('addTempRow', { type, name, status })
```
Comment (alaister) [inline studio/localStores/storageExplorer/StorageExplorerStore.js:1579]: cheeky `console.log()` sneaking in

**Review by alaister — APPROVED** [review]
LGTM!

One nit though is I can't hit escape when renaming a file.

**Comment by joshenlim** [comment]
> One nit though is I can't hit escape when renaming a file.

ahh i dont think this was implemented actually, but lemme add that in within this PR

**Review by alaister — APPROVED** [review]
Double approve

**Pull request merged**